### ISSUE TITLE
Don't work on some Xiaomi device

### ISSUE THREAD

**Issue by Aibeto** [Briclyaz/NLSound_module_QCom#10]
[magisk_install_log_2024-06-07T21.49.51.log](https://github.com/user-attachments/files/15747011/magisk_install_log_2024-06-07T21.49.51.log)
I tried many times with different configurations, attached is an example log, I actually still get the original sample rate and bit depth (48k/16bit) after rebooting the device.
Xiaomi Pad 6 Pro
hyperOS 1.0.5.0

**Comment by Briclyaz** [comment]
Hi! What playback source did you check? Many games, for example, are limited to 16-bit and 48000 Hz only. The module sets the parameters loyally, with the ability to lower both bit rate and sample rate if required by the application to prevent conflicts.

**Comment by Aibeto** [comment]
No, the module doesn't actually work at all, the audio output is almost exactly the same as it was before the module was installed

**Comment by Briclyaz** [comment]
Well, need dump files from your rom.
